sickbeard/helpers.py

```diff
@@ -595,8 +593,8 @@ def moveAndSymlinkFile(srcFile, destFile):
         ek(shutil.move, srcFile, destFile)
         fixSetGroupID(destFile)
         ek(symlink, destFile, srcFile)
-    except Exception:
-        logger.log(u"Failed to create symlink of " + srcFile + " at " + destFile + ". Copying instead", logger.ERROR)
+    except Exception as e:
+        logger.log(u"Failed to create hardlink of %s at %s. Error: %r. Copying instead" % (srcFile, destFile, ex(e)),logger.WARNING)
```

Comment:
I didnt created them. Only fixed. We can remove some. Dont mind.

Thoses errors are not really errors to user submit.
What should I do then?